### DISCUSSION TITLE
v21.1.2 ComponentBuilder storybook file location

### DISCUSSION THREAD

**Discussion by bewards** [Sitecore/jss#1550] [Help]
### Description

With the upgrade to ComponentBuilder plugin replacing the legacy component factory, is there currently a way to keep our stories files in the components folder?

Previously, we had been running our story files side-by-side with our component files. The legacy component factory had the ability to filter out files from the component factory generator, specifically in the \scripts\generate-component-factory.ts. I’ve not found a way to do the same thing with ComponentBuilder.

**Comment by art-alexeyenko** [comment]
Hi @bewards ! 

Sometimes it feels like our community thinks of similar things at the same time, we've had a report for a similar problem just recently here:
https://github.com/Sitecore/jss/discussions/1551

We have a task in our backlog to add filtering logic for the builder, so that unwanted files or components are not registered. Thank you for reporting this.
